massa-node/base_config/config.toml

```diff
@@ -156,6 +156,10 @@
     # Max number of in connection overflowed managed by the handshake
     # that send a list of peers
     max_in_connection_overflow = 100
+    # Read limitation for a connection in bit by 0.100 milliseconds
```

Comment:
I think the units are "bytes per second". If it's the case, those values should be adjusted in consequence